server/src/child.rs

```diff
@@ -127,10 +132,18 @@ struct TargetState {
 
 struct TraceState {
     execution: Option<Execution>,
+    exec: Option<Exec>,
     event: Option<debug::Event>,
     attached: bool,
 }
 
+#[derive(Clone)]
+pub struct Exec {
+    pub e_type: String,
+    pub status: String,
+    pub execution_time: i32,
+}
+
```

Comment:
Shouldn't need to track this stuff here, since the server thread already knows what type of execution it's requesting. Removing this type should also get rid of a lot of code below in this file and in `main.rs`.